### PR TITLE
Added Spinning Loader

### LINKED ISSUE #3: Spinning Loader
Add a spinning loader inside Loaders component.

### PR DESCRIPTION
**Pull request by rishicds**
# Fixes Issue🛠️

Closes #3 

# Description👨‍💻 

Added Spinning Loader

# Type of change📄
- [X] New feature (non-breaking change which adds functionality)

# Checklist✅ 

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added demonstration in the form of GIF/video file
- [X] I am an Open Source Contributor

# Screenshots/GIF📷
image

### CODE REVIEW

File: assets/html_files/loaders.html [hunk 1]
```diff
@@ -42,6 +42,19 @@ <h1>Wifi Loader</h1>
...
+                    <a href="../../Components/Loaders/Spinning-Loader/index.html" title="Live Preview" target="_blank">
+                        <img src="../images/link.png">
+                    </a>
+                </div>
+                <div class="source">
+                    <a href="https://github.com/Rakesh9100/Beautiify/tree/main/Components/Loaders/Spinning-Loader" title="Source Code" target="_blank">
+                        <img src="../images/github.png">
+                    </a>
+                </div>
+
```
Comment (Rakesh9100) [inline assets/html_files/loaders.html:57]: Include a ending div tag here as it is missing from your section @rishicds

Comment (rishicds) [reply assets/html_files/loaders.html]: hello, I have added the tag @Rakesh9100

**Comment by rishicds** [comment]
Hello, I have fixed it, please check it out @Rakesh9100

**Comment by Rakesh9100** [comment]
Nice, Merged!! 🎉🥳 Keep Contributing @rishicds

**Pull request merged**